Library/Homebrew/cask/artifact/abstract_artifact.rb

```diff
@@ -148,6 +149,41 @@ def initialize(cask, *dsl_args)
         @dsl_key = nil
         @english_article = nil
         @english_name = nil
+        @sort_order = [
+          PreflightBlock,
+          # The `uninstall` stanza should be run first, as it may
```

Comment:
Would be good to avoid repeating all of this

Reply:
@MikeMcQuaid 
It would be better to declare this class array as a constant, how about?